docs/DeveloperGuide.adoc

```diff
@@ -337,30 +364,257 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. System shows an error message.
 +
 Use case resumes at step 2.
 
```

Comment:
The use case is quite good. You can improve your use case by considering * extensions.